org.tenacityaudio.Tenacity.yaml

```
@@ -0,0 +1,176 @@
+    directory: extensions/Plugins
+    version: '22.08'
+    add-ld-path: lib
+    merge-dirs: ladspa;lv2;lxvst
+    subdirectories: true
+    no-autodownload: true
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/aclocal
+  - /share/man
+  - '*.la'
+  - '*.a'
+modules:
+
+  - name: wxwidgets
+    rm-configure: true
+    config-opts:
+      - --with-libpng
+      - --with-zlib
+      - --with-cxx=17
+      - --disable-sdltest
+      - --disable-webview
+      - --disable-webviewwebkit
+      - --disable-ribbon
+      - --disable-propgrid
+      - --disable-richtext
+      - --with-expat=builtin
+      - --with-libiconv=/usr
+    cleanup:
+      - /share/bakefile
+    sources:
+      - type: git
+        url: https://github.com/wxWidgets/wxWidgets
+        tag: v3.2.1
+        commit: 97e99707c5d2271a70cb686720b48dbf34ced496
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+
+  - deps/lame-3.100.json
+
+  - name: libid3tag
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://codeberg.org/tenacityteam/libid3tag.git
+        tag: 0.16.2
+        commit: 3df1882b76c2f44a274af1af33497125c95679b9
+        x-checker-data:
+          type: git
+          tag-pattern: ^([\d.]+)$
+
+  - shared-modules/libmad/libmad.json
+  - shared-modules/linux-audio/jack2.json
+  - shared-modules/linux-audio/lv2.json
+  - shared-modules/linux-audio/lilv.json
+  - deps/suil.json
+
+  - name: portaudio
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/PortAudio/portaudio.git
+        tag: v19.7.0
+        commit: 147dd722548358763a8b649b3e4b41dfffbcfbb6
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+
+  - name: portsmf
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/tenacityteam/portsmf.git
+        tag: '239'
+        commit: 951b636f7d0cba370d483a91f1897c71f3d98530
+        x-checker-data:
+          type: git
+          tag-pattern: ^([\d.]+)$
+
+  - name: portmidi-mixxxdj
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/mixxxdj/portmidi.git
+        tag: '236'
+        commit: fda97c3be2baf10438574d10bb2a6b8b4f1140c4
+        x-checker-data:
+          type: git
+          tag-pattern: ^([\d.]+)$
+
+  - name: soxr
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/chirlu/soxr.git
+        tag: 0.1.3
+        commit: 945b592b70470e29f917f4de89b4281fbbd540c0
+        x-checker-data:
+          type: git
+          tag-pattern: ^([\d.]+)$
+
+  - name: libsbsms
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/claytonotey/libsbsms.git
+        tag: 2.3.0
+        commit: e99cd7e6c6367e476577be34d2fdbe2023904d7e
+        x-checker-data:
+          type: git
+          tag-pattern: ^([\d.]+)$
+
+  - name: vamp-plugin-sdk
+    sources:
```

```suggestion
    no-parallel-make: true,
    sources:
```

it's buggy otherwise.